mozilla/fx-private-relay
# change from drf-yasg to drf-spectacular

This PR fixes the CI failure in https://github.com/mozilla/fx-private-relay/pull/3547 by switching from `drf-yasg` to `drf-spectacular`.

How to test:
1. Set `API_DOCS_ENABLED=True`
2. Go to http://127.0.0.1:8000/api/v1/docs/
   * [ ] You should still see the Swagger UI docs
3. Go to http://127.0.0.1:8000/api/v1/docs/redoc/
   * [ ] You should see the NEW Redocly docs!
